doc/decoding/decoding_intro.rst

```diff
@@ -463,13 +463,13 @@ We can try Fisher's `Linear Discriminant Analysis (LDA)
 
 Import the module::
 
-    >>> from sklearn.lda import LDA
+    >>> from sklearn.lda import LDA  # doctest: +SKIP
```

Comment:
With recent sklearn, LDA can be imported through.
```from sklearn.discriminant_analysis import LinearDiscriminantAnalysis```

Shall I add another line suggesting import for recent sklearn ?